sdk/openai/Azure.AI.OpenAI/tests/OpenAIInferenceTests.cs

```diff
@@ -54,5 +65,18 @@ public async Task EmbeddingTest()
             var response = await client.GetEmbeddingsAsync(EmbeddingsDeploymentId, embeddingsRequest);
             Assert.That(response, Is.InstanceOf<Response<Embeddings>>());
         }
+
+        /// <summary>
+        /// Test Exception throw.
+        /// </summary>
+        [RecordedTest]
+        public void RequestFailedExceptionTest()
+        {
+            var client = GetClient();
+            CompletionsOptions completionsRequest = new CompletionsOptions();
+            completionsRequest.Prompt.Add("Hello world");
+            Assert.That(async () => await client.GetCompletionsAsync("BAD_DEPLOYMENT_ID", completionsRequest),
```

Comment:
nit: should we assert the code that we expect to come back?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/33789